VirtoCommerce.Storefront/Startup.cs

```diff
@@ -89,6 +91,22 @@ public void ConfigureServices(IServiceCollection services)
 
             services.Configure<StorefrontOptions>(Configuration.GetSection("VirtoCommerce"));
 
+            if (string.Equals(
```

Comment:
But we already has `ConfigureWebHostDefaults` in Program.cs. I think it should work when `ASPNETCIRE_FORWARDEDHEADERS_ENABLED` specified without this code, out of the box:
https://stackoverflow.com/a/58908072
https://github.com/dotnet/aspnetcore/blob/1480b998660d2f77d0605376eefab6a83474ce07/src/DefaultBuilder/src/WebHost.cs#L240